azext_iot/common/_azure.py

```diff
@@ -120,7 +120,9 @@ def _find_iot_dps_from_list(all_dps, dps_name):
 
 
 def get_iot_central_tokens(cmd, app_id, token, central_dns_suffix):
-    import requests
+    from requests import post
```

Comment:
@digimaun - does this need to be done for the hub monitor events too?

Reply:
I am not aware of a requirement for this functionality in hub monitor-events. So I'm leaning towards no in this PR. We also need to verify `uamqp` supports the functionality of a TLS connection without verification.